### PR TITLE
fix: set default region

### PR DESCRIPTION
**Pull request by brivu**
This `PR` updates the descriptions of the `configure_default_region` and the `configure_profile_region` parameters. It also removed the `--profile` flag from the `aws configure set default.region` cli command in the `configure.sh` script. Lastly, it adds testing and validation for configuring both the default region and the profile region. 

Currently, there's some confusion regarding the configuration of the `default region` versus the `profile region`. 

According to the `AWS` Documentation: 

**`aws configure set default.region`**:
 
- This command sets the default region for the AWS CLI. The specified region will be used by default for all AWS CLI commands unless overridden.
-The value set using default.region is stored in the AWS CLI configuration file (`~/.aws/config`), specifically in the `[default]` section.
-Example usage: `aws configure set default.region us-west-2`

**`aws configure set region`**:

- This command sets the region for the current profile in the AWS CLI configuration. The specified region will be used for commands executed under that specific profile.
-The value set using region is stored in the AWS CLI configuration file (`~/.aws/config`) within the specific profile section.
-Example usage: `aws configure set region us-west-2 --profile myprofile`

In summary, `aws configure set default.region` sets the default region for all commands executed by the AWS CLI, while `aws configure set region` sets the region for a specific profile.

### CODE REVIEW

File: src/commands/setup.yml [hunk 1]
```diff
@@ -64,12 +64,14 @@ parameters:
   configure_default_region:
     description: |
       Some AWS actions don't require a region; set this to false if you do not want to store a default region in ~/.aws/config
+      Any AWS cli command will default to this region if none is specified with the --region cli parameter.
```
Comment (EricRibeiro) [inline src/commands/setup.yml:67]:
```suggestion
      Any AWS CLI command will default to this region if none is specified with the --region CLI parameter.
```

File: src/commands/setup.yml [hunk 2]
```diff
@@ -64,12 +64,14 @@ parameters:
...
     type: boolean
     default: true
 
   configure_profile_region:
     description: |
-      Boolean whether to configure the region for the custom (non-default) profile or not
+      Boolean whether to configure the region for the custom (non-default) profile. The specified region will be used for AWS cli
+      commands executed under that specific profile using the --profile cli parameter.
```
Comment (EricRibeiro) [inline src/commands/setup.yml:74]:
```suggestion
      Boolean whether to configure the region for the custom (non-default) profile. The specified region will be used for AWS CLI
      commands executed under that specific profile using the --profile CLI parameter.
```

File: src/scripts/configure.sh [hunk 3]
```diff
@@ -7,7 +7,7 @@ fi
 
 ORB_ENV_ACCESS_KEY_ID=$(circleci env subst "\$$ORB_ENV_ACCESS_KEY_ID")
 ORB_ENV_SECRET_ACCESS_KEY=$(circleci env subst "\$$ORB_ENV_SECRET_ACCESS_KEY")
-ORB_EVAL_AWS_CLI_REGION=$(circleci env subst "\$$ORB_EVAL_AWS_CLI_REGION")
+ORB_EVAL_AWS_CLI_REGION=$(circleci env subst "$ORB_EVAL_AWS_CLI_REGION")
```
Comment (EricRibeiro) [inline src/scripts/configure.sh:10]: If this is an `EVAL` variable, shouldn't we use `eval echo` instead of `circleci env subst`?

Since it's not a directory, we can make it an "STR" instead.

**Review by EricRibeiro — CHANGES_REQUESTED** [review]

File: src/scripts/configure.sh (hunk 3 above)
Comment (brivu) [reply src/scripts/configure.sh:10]: `PR`   #141  contains all the renaming of variables. Once that's merged, I'll deal with the conflicts and make the changes here.

**Review by KyleTryon — APPROVED** [review]

**Review by EricRibeiro — APPROVED** [review]
